bftengine/src/bcstatetransfer/BCStateTran.hpp

```diff
@@ -273,10 +273,10 @@ class BCStateTran : public IStateTransfer {
   ///////////////////////////////////////////////////////////////////////////
   const Config config_;
   const set<uint16_t> replicas_;
-  const uint32_t maxVBlockSize_;
-  const uint32_t maxItemSize_;
-  const uint32_t maxNumOfChunksInAppBlock_;
-  const uint32_t maxNumOfChunksInVBlock_;
+  uint32_t maxVBlockSize_;
+  uint32_t maxItemSize_;
+  uint32_t maxNumOfChunksInAppBlock_;
+  uint32_t maxNumOfChunksInVBlock_;
```

Comment:
Initialize those data members here or in the ctor.

Reply:
Done